src/messages/Message.hpp

```diff
@@ -47,6 +47,7 @@ struct Message {
     QString loginName;
     QString displayName;
     QString localizedName;
+    QString userID;
```

Comment:
This should be added to the JSON serialization (the test snapshots need to be updated afterwards as well).

Reply:
Added, I haven't done anything for the pubsub messages though. So those tests' have their userID field empty.